server.js

```diff
@@ -0,0 +1 @@
+// Hi blogApp developer
```

Comment:
remove console from blogValidator

Reply:
Always try and seggregate code into controller and service wherein service contains your business logic

Reply:
remove unnecessary console from user-model file

Reply:
remove unnecessary comments from validate-middleware